src/pageobjects/sidebar/ViewContent.ts

```diff
@@ -50,7 +50,8 @@ export class ViewContent extends BasePage<typeof ViewContentLocators> {
 
         const availableSections: Set<string> = new Set()
         for (const element of elements) {
-            const sectionTitle = await this.sectionTitle$.getText()
+            const sectionTitle = await (await element.$(this.locators.sectionTitle)).getText()
```

Comment:
We can chain the promise calls here:
```suggestion
            const sectionTitle = await element.$(this.locators.sectionTitle).getText()
```

Reply:
sure. thx. shall I just use the "Commit suggestion" feature or rewrite the commit?

Reply:
ok. done